### DISCUSSION TITLE
Force generation of builders for models

### DISCUSSION THREAD

**Discussion by lmartella1** [quarkiverse/quarkus-openapi-generator#570] [Q&A]
Hello,
Is there a way to instrument the client code generation from OpenAPI specification to generate builders for models as available by setting _generateBuilders=true_ in [jaxrs-spec OpenAPI generator](https://openapi-generator.tech/docs/generators/jaxrs-spec)?
Thanks,
Luca

**Comment by ricardozanini** [comment]
Hi Luca! At the moment, we don't have this feature. But I believe it won't be too hard to implement having the right template changes. Are you willing to send a PR?
